assets/css/components/_app-header.scss

```
@@ -3,7 +3,7 @@
     padding: rem(10px);
   }
 
-  .user-setting,
+  .setting-nav-link,
```

Can we have `navbar-link` similar to `navbar-text`? 

```suggestion
  .navbar-link,
```

or other links do not need this styling, only for settings? 🤔